cmd/bridge/config/auth/authoptions.go

```diff
@@ -32,6 +33,7 @@ type AuthOptions struct {
 	ClientSecretFilePath string
 	CAFilePath           string
 	OCLoginCommand       string
+	BearerToken          string
```

Comment:
I'd call it `StaticUserBearerToken`. Add a comment that explains that it'll override the authenticator and is supposed to be used for development purposes only.